experimental/packages/exporter-trace-otlp-grpc/package.json

```diff
@@ -49,8 +49,8 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@grpc/proto-loader": "^0.7.3",
-    "@opentelemetry/api": "^1.0.0",
-    "@opentelemetry/otlp-exporter-base": "0.35.1",
+    "@opentelemetry/api": "1.4.1",
```

Comment:
seems not only examples are effected by this api dependency change.

Reply:
Not sure if it is actually a problem.
For dev dependencies `1.4.1` is actually the same as we anyway test the current, linked version nothing from NPM.